### PR TITLE
fix: added empty object to shutdown method

### PR DESCRIPTION
**Pull request by janrxyz**

### CODE REVIEW

**Review by blu3beri — APPROVED** [review]

File: android/src/main/java/com/reactnativebledidcomm/peripheral/PeripheralManager.kt [hunk 1]
```diff
@@ -63,8 +63,12 @@ class PeripheralManager(
     }
 
     fun shutdownPeripheral() {
-        this.gattServer.connectedDevices.clear()
+        val connectedDevices = this.bluetoothManager.getConnectedDevices(BluetoothProfile.GATT)
+        for (device in connectedDevices) {
+            this.gattServer.cancelConnection(device)
+        }
         this.gattServer.close()
```
Comment (blu3beri) [inline android/src/main/java/com/reactnativebledidcomm/peripheral/PeripheralManager.kt:70]: Shouldn't we close the gattServer when we stop advertising?

File: android/src/main/java/com/reactnativebledidcomm/BleDidcommModule.kt [hunk 2]
```diff
@@ -54,6 +54,7 @@ class BleDidcommModule(private val context: ReactApplicationContext) :
 
     @ReactMethod
     fun shutdownCentral(
+        @Suppress("UNUSED_PARAMETER") options: ReadableMap,
```
Comment (blu3beri) [inline android/src/main/java/com/reactnativebledidcomm/BleDidcommModule.kt:57]: Central needs no changes?

**Review by blu3beri — APPROVED** [review]
Has this been tested with:

1. Create connection
2. Send message (arrives)
3. Disconnect
4. Send message (error)

for both sides?

File: android/src/main/java/com/reactnativebledidcomm/peripheral/PeripheralManager.kt (hunk 1 above)
Comment (janrxyz) [reply android/src/main/java/com/reactnativebledidcomm/peripheral/PeripheralManager.kt:70]: I've moved the close to the `stopAdvertising` function.

File: android/src/main/java/com/reactnativebledidcomm/BleDidcommModule.kt (hunk 2 above)
Comment (janrxyz) [reply android/src/main/java/com/reactnativebledidcomm/BleDidcommModule.kt:57]: Do you mean the same changes as in the `peripheralManager`? The central shutdown works fine without those changes.

**Comment by TimoGlastra** [comment]
Why do we need the empty object?

File: android/src/main/java/com/reactnativebledidcomm/BleDidcommModule.kt (hunk 2 above)
Comment (blu3beri) [reply android/src/main/java/com/reactnativebledidcomm/BleDidcommModule.kt:57]: Not the same changes per se. As long as it is tested with the flow I mentioned above it is fine.

**Comment by blu3beri** [comment]
> Why do we need the empty object?

I am not sure on the internals, tried finding information about it before, but a function without any params needs an empty object to work. It might be fixable by tweaking the Objective-C interface, but this seems to be a fine solution to me.

**Comment by TimoGlastra** [comment]
Can we add a comment? I would remove this I it wouldn't have a comment 😉

**Pull request merged**